heroic-core/src/main/java/com/spotify/heroic/CoreQueryManager.java

```diff
@@ -278,20 +282,26 @@ public Group(List<ClusterShard> shards) {
                 .orElseGet(() -> features.withFeature(Feature.END_BUCKET, () -> BucketStrategy.END,
                     () -> BucketStrategy.START));
 
-            final AggregationCombiner combiner;
-
+            if (aggregationInstance instanceof GroupInstance) {
+                if (((GroupInstance) aggregationInstance).getEach() instanceof TdigestInstance) {
+                    metricType = MetricType.DISTRIBUTION_POINTS;
+                }
+            }
+            if (aggregationInstance instanceof TdigestInstance) {
```

Comment:
This is not necessary. TdigestPoint is used in the final stage of aggregation to compute stat.